src/main/scala/com/workday/warp/persistence/mysql/HasWarpSlickDsl.scala

```diff
@@ -52,6 +52,72 @@ trait HasWarpSlickDsl {
       expression.apply(timestamp)
     }
 
+    /**
+      *  Translates to YEAR() function
+      *  @return just the year in Int
+      */
+    def year(): Rep[Int] = {
+      val expression = SimpleExpression.unary[Timestamp, Int] { (timestamp, queryBuilder) =>
+        queryBuilder.sqlBuilder += " YEAR ("
+        queryBuilder.expr(timestamp)
+        queryBuilder.sqlBuilder += ")"
+      }
+      expression.apply(timestamp)
+    }
+
+    /**
+      * Translates to DATE() function
+      * @return just date in "yyyy-MM-dd"
+      */
+    def date(): Rep[String] = {
+      val expression = SimpleExpression.unary[Timestamp, String] { (timestamp, queryBuilder) =>
+        queryBuilder.sqlBuilder += " DATE ("
+        queryBuilder.expr(timestamp)
+        queryBuilder.sqlBuilder += ")"
+      }
+      expression.apply(timestamp)
+    }
+
+    /**
+      * UNIX_TIMESTAMP operation - seconds elapsed since 1970-01-01 00:00:00 UTC
+      * @return seconds in int
+      */
+    def UNIX_TIMESTAMP(): Rep[Int] = {
+      val expression = SimpleExpression.unary[Timestamp, Int] { (timestamp, queryBuilder) =>
+        queryBuilder.sqlBuilder += " UNIX_TIMESTAMP ("
+        queryBuilder.expr(timestamp)
+        queryBuilder.sqlBuilder += ")"
+      }
+      expression.apply(timestamp)
+    }
+
+    /**
+      * subdate operation: date - interval
+      * @return timestamp date
+      */
+    def subdate(date: String, interval: String): Rep[String] = {
+      val expression = SimpleExpression.unary[String, String] { (date, queryBuilder) =>
+        queryBuilder.sqlBuilder += " subdate("
+        queryBuilder.expr(date)
+        queryBuilder.sqlBuilder += ", INTERVAL "
+        queryBuilder.sqlBuilder += interval
+        queryBuilder.sqlBuilder += ")"
+      }
+      expression.apply(date)
+    }
   }
 
+  object TimeStampExtensions {
+
+    /**
+      * equivalent to NOW()
+      * @return string of date and time
+      */
+    def now(): Rep[String] = {
+      val expression = SimpleExpression.nullary[String] { (queryBuilder) =>
+        queryBuilder.sqlBuilder += " NOW() "
+      }
+      expression
```

Comment:
Do you need the `expression` val here? There's an argument for consistency with methods above, but you're not calling `.apply(...)` on it so you could feasibly just return the expression without putting it into a val first